container.go

```diff
@@ -57,6 +57,18 @@ type ContainerStatus struct {
 	Annotations map[string]string
 }
 
+// ContainerResources describes container resources
+type ContainerResources struct {
+	Resources
```

Comment:
Why do you need this ? The PodConfig already have this info.

Reply:
to know container's resources, and remove only those resources on stop